extension_helpers/_distutils_helpers.py

```diff
@@ -10,9 +10,10 @@
 
 import os
 import sys
-from distutils import ccompiler
-from distutils.dist import Distribution
-from distutils.errors import DistutilsError
+
+from setuptools.command.build_ext import new_compiler
+from setuptools.dist import Distribution
+from setuptools.errors import DistutilsError
 
```

Comment:
I assume none of these imports are new(ish) to setuptools and they have been providing versions of this for ages?

Should we have a test which checks our stuff works with a minimum version of setuptools?

Reply:
Yes sounds good! Will add.